### DISCUSSION TITLE
How do I install a tree-sitter grammar that is not part of NVChad?

### DISCUSSION THREAD

**Discussion by etrigan63** [NvChad/NvChad#2312] [Q&A]
I want to install a `tree-sitter` grammar for `hyprland` but cannot figure out how without borking the whole thing. Any constructive guidance is appreciated.

https://github.com/luckasRanarison/tree-sitter-hypr

**Comment by siduck** [comment]
@etrigan63 just like how you'd do with non nvchad config :/

**Reply by etrigan63** [reply]
@siduck Telling me to do something I don't know how to do, does not answer the question.

**Reply by etrigan63** [reply]
![Screenshot_2023-08-19-23-19-35_7444](https://github.com/NvChad/NvChad/assets/7347264/33f0f77f-420a-4a71-b548-7dadef80fac3)
Which of these options would I use?
